Resources/config/sitemap.xml

```diff
@@ -7,12 +7,6 @@
 
     <parameters>
         <parameter key="cmf_seo.sitemap.controller.class">Symfony\Cmf\Bundle\SeoBundle\Controller\SitemapController</parameter>
-        <parameter key="cmf_seo.sitemap.provider.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\Provider</parameter>
-
-        <parameter key="cmf_seo.sitemap.loader_chain.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\Loader</parameter>
-        <parameter key="cmf_seo.sitemap.guesser.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\Guesser</parameter>
-        <parameter key="cmf_seo.sitemap.voter_chain.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\Voter</parameter>
-        <parameter key="cmf_seo.sitemap.phpcr.public_workflow_voter.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\PublishWorkflowUrlInformationVoter</parameter>
```

Comment:
having configuration for class is deprecated. imho we should not add new ones anymore. (but keep existing until a major version bump to avoid BC breaks)